DEPLOYMENT_CHECKLIST.md

```diff
@@ -1,108 +1,74 @@
 # Deployment Checklist
 
-This checklist is seen as a guide to deploy the stack to a new chain.
+This checklist is seen as a guide to deploy the contracts to a new chain.
 
 ## Pre-Deployment
 
+- [ ] Run `yarn` in the repository root to install the dependencies.
+- [ ] Run `yarn build` in `packages/contracts` to make sure the contracts compile.
+- [ ] Run `yarn test` in `packages/contracts` to make sure the contract tests succeed.
+  - In order to run the tests on localhost, add the following in the packages/contracts/.env
+    ```js
+    HARDHAT_DAO_ENS_DOMAIN = testdao.eth;
+    HARDHAT_PLUGIN_ENS_DOMAIN = testpluginrepo.eth;
+    ```
```

Comment:
```suggestion
    ```
    HARDHAT_DAO_ENS_DOMAIN = "testdao.eth"
    HARDHAT_PLUGIN_ENS_DOMAIN = "testpluginrepo.eth"
    ```
```
This is not js code.